ci/cpu/upload.sh

```diff
@@ -22,32 +22,24 @@ if [[ -z "$MY_UPLOAD_KEY" ]]; then
   return 0
 fi
 
-################################################################################
-# SETUP - Get conda file output locations
-################################################################################
-
-gpuci_logger "Get conda file output locations"
-
-export LIBCUML_FILE=`conda build --no-build-id --croot ${CONDA_BLD_DIR} conda/recipes/libcuml --output`
-export CUML_FILE=`conda build --croot ${CONDA_BLD_DIR} conda/recipes/cuml --python=$PYTHON --output`
-
 ################################################################################
 # UPLOAD - Conda packages
 ################################################################################
 
 gpuci_logger "Starting conda uploads"
 
 if [[ "$BUILD_LIBCUML" == "1" && "$UPLOAD_LIBCUML" == "1" ]]; then
-  test -e ${LIBCUML_FILE}
-  echo "Upload libcuml"
-  echo ${LIBCUML_FILE}
-  gpuci_retry anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-rapidsai} ${LABEL_OPTION} --skip-existing ${LIBCUML_FILE} --no-progress
+  LIBCUML_FILES=$(conda build --no-build-id --croot ${CONDA_BLD_DIR} conda/recipes/libcuml --output)
+  while read -r LIBCUML_FILE; do
+    test -e ${LIBCUML_FILE}
+    echo "Upload libcuml file: ${LIBCUML_FILE}"
+    gpuci_retry anaconda -t ${MY_UPLOAD_KEY} upload -u ${CONDA_USERNAME:-rapidsai} ${LABEL_OPTION} --skip-existing ${LIBCUML_FILE} --no-progress
+  done <<< "${LIBCUML_FILES}"
```

Comment:
From the comment below, we should be able to remove this loop and just call `anaconda upload <file1> <file2> ...`. I'm going to test this locally and see if it works. If so, I'll update the `cudf` PR and we should probably update these other PRs as well.


- https://github.com/rapidsai/cudf/pull/10326#discussion_r830010035

Reply:
Updated in this commit: https://github.com/rapidsai/cudf/pull/10326/commits/7deedf6089144c4a9db5656dab2aed242b6246a8